src/main/java/org/jitsi/videobridge/cc/BitrateController.java

```diff
@@ -1385,6 +1359,11 @@ else if (encoding.getHeight() <= Config.thumbnailMaxHeightPx())
                     ratesList.add(
                         new RateSnapshot(encodingBitrateBps, encoding));
                 }
+
+                if (encoding.getHeight() <= videoConstraints.getPreferredHeight())
```

Comment:
This could probably use a comment, explaining that we jump immediately to the preferred encoding.